tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -736,14 +736,13 @@ Then('the collaborators list for file/folder/resource {string} should be empty',
   assert.strictEqual(count, 0, `Expected to have no collaborators for '${resource}', Found: ${count}`)
 })
 
-Then('the file/folder/resource {string} should be in {string} state on the webUI', function (filename, status) {
-  status = status === 'Accepted' ? '' : status
-  return client.page.sharedWithMePage().assertDesiredStatusIsPresent(filename, status)
-})
-
-Then('file/folder {string} shared by {string} should be in {string} state on the webUI', function (element, user, status) {
-  status = status === 'Accepted' ? '' : status
-  return client.page.sharedWithMePage().assertDesiredStatusIsPresent(element, status, user)
+Then('file/folder {string} shared by {string} should be in {string} state on the webUI', async function (resource, sharer, expectedStatus) {
+  const currentStatus = await client.page.sharedWithMePage().getShareStatusOfResource(resource, sharer)
+  return assert.strictEqual(
+    currentStatus,
+    expectedStatus,
+    `Expected resource '${resource}' shared by '${sharer}' to be present with status '${expectedStatus}' but got found: '${currentStatus}'!`
```

Comment:
```suggestion
    `Expected resource '${resource}' shared by '${sharer}' to be in state '${expectedStatus}' but it is in state: '${currentStatus}'!`
```

please also make line shorter

similar for the other steps

Reply:
done! :rocket: